jvm-diagnostics/src/main/java/com/palantir/jvm/diagnostics/JvmDiagnostics.java

```diff
@@ -122,6 +124,20 @@ public static Optional<CpuSharesAccessor> cpuShares() {
         }
     }
 
+    /**
+     * Returns a {@link HotspotDnsCacheTtlAccessor}. This functionality is not supported on all java runtimes,
+     * and an {@link Optional#empty()} is returned in cases cpu share information is not supported.
```

Comment:
nit: not sure what cpu share info has to do the the DNS cache info

Reply:
Ah bad copy paste, thanks!